docs/components/popover/README.md

```diff
@@ -87,14 +87,14 @@ element via the `<b-popover>` component or `v-b-popover` directive.
     },
     methods: {
       onCancel() {
-        this.$parent.$emit('close');
+        this.$refs.popover.$emit('close');
```

Comment:
Yeah..  I kept thinking app was a child of b-popover, but its not...  good catch!